### PR TITLE
Terra Scripts for MVP Optimus Processing

### PR DESCRIPTION
**Pull request by jessicaway**

### CODE REVIEW

File: pipelines/skylab/optimus/scripts/parse_terra.py [hunk 1]
```diff
@@ -0,0 +1,121 @@
+import argparse
```
Comment (jessicaway) [inline pipelines/skylab/optimus/scripts/parse_terra.py:1]: @kishorikonwar Is this the same script (more or less) as the "parse_terra_tsv.py" script that is already checked in? Let's not have the same script checked in twice please. Also I think the other one is more up-to-date, can we please make sure the workflow-controller calls the right one?

File: pipelines/skylab/optimus/scripts/fiss.ipynb [hunk 2]
```diff
@@ -0,0 +1,307 @@
+{
```
Comment (jessicaway) [inline pipelines/skylab/optimus/scripts/fiss.ipynb:1]: Is this file needed? If so is it ready to be checked-in?

File: pipelines/skylab/optimus/scripts/json_templates.py [hunk 3]
```diff
@@ -0,0 +1,30 @@
+optimus_inputs = {
```
Comment (jessicaway) [inline pipelines/skylab/optimus/scripts/json_templates.py:1]: Do we need this file?

File: pipelines/skylab/optimus/scripts/workspace_controller.py [hunk 4]
```diff
@@ -0,0 +1,316 @@
+import argparse
```
Comment (jessicaway) [inline pipelines/skylab/optimus/scripts/workspace_controller.py:1]: We will put a ticket in the backlog to clean this code up at some point

**Review by nikellepetrillo — APPROVED** [review]

**Review by kishorikonwar — APPROVED** [review]
